src/components/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please 🙏🏽 don't disable linters, instead of fixing them manually, it will help you to use better practices of JS.